### ISSUE TITLE
Dropdown menus - option selection

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#967]
**Problematic behavior**
In dropdown menus, the selected choice is not announced by screen readers.

**Expected behavior/code**
The selection made in dropdown menus must be announced to screen readers.

**Steps to Reproduce**

1. Open a document
2. Configure document sharing
3. Select a user level and that's it.

**Possible Solution**
Hide the icon text with the `aria-hidden="true"` attribute, add hidden text with the `visually-hidden` class. All information can be found here: https://www.ffoodd.fr/masquage-accessible-de-pointe/

```
<button role="menuitem" aria-label="Privé" class="sc-95e747a0-0 iPUaJj --docs--box-button "><div class="sc-95e747a0-0 jXQfiJ"><span class="sc-95e747a0-0 sc-aebf9ca2-0 iimkVR bUfZGn --docs--icon-bg material-icons" aria-hidden="true">lock</span><span class="sc-95e747a0-0 sc-aebf9ca2-0 iimkVR hPUcxb">Privé</span></div><span class="sc-95e747a0-0 sc-aebf9ca2-0 iimkVR jxFArT --docs--icon-bg material-icons" aria-hidden="true">check</span><span class="visually-hidden">Coché</span></button>
```
**Additional context/Screenshots**

<img width="302" alt="Image" src="https://github.com/user-attachments/assets/9164d958-3c24-4d6c-afc5-c00f5d92071a" />

**Comment by Ovgodd** [comment]
This issue has been resolved in another pull request.

https://github.com/user-attachments/assets/2d808fb2-24ce-4fb4-8cc6-5a8add00bc7b
